testcases/OpTestHMIHandling.py

```diff
@@ -350,10 +360,13 @@ def _test_malfunction_allert(self):
 
         # Core checkstop will lead to system IPL, so we will wait for certain time for IPL
         # to finish
-        #l_res = self.cv_SYSTEM.sys_get_ipmi_console().run_command(l_cmd, timeout=600)
-        console = self.cv_SYSTEM.sys_get_ipmi_console()
+        console = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
+        res = console.run_command("uname -a") # perform any command to make sure console is logged in
```

Comment:
It is using ssh connection, login to SSH connection won't confirm the login to console based session.

Reply:
> It is using ssh connection, login to SSH connection won't confirm the login to console based session.

Can you clarify this comment or provide recreate ?

Reply:
> clarification - We can probably clean up the modules further to properly only reference the intended object.  Once this patch is accepted I can work on another pass to cleanup references to the ".sol" code.

common/OpTestSSH.py        
# Users expecting "Host IPMI" will reference console.sol so make it available
        self.sol = self.console

Reply:
Yeah, we need to use sol console instead of SSH session, see below in one of the comments i pasted the diff with right fix.